### ISSUE TITLE
Symbol not found: ..._Pathszumattermostzuapi_version1_closure

### ISSUE THREAD

**Issue by jtdaugherty** [matterhorn-chat/matterhorn#154]
Behold:
```
Configuring matterhorn-0.1.0.0...
Preprocessing executable 'matterhorn' for matterhorn-0.1.0.0...
[ 1 of 34] Compiling Completion       ( src/Completion.hs, /Users/cygnus/src/matterhorn/dist-newstyle/build/matterhorn-0.1.0.0/build/matterhorn/matterhorn-tmp/Completion.o )
[ 2 of 34] Compiling Paths_matterhorn ( /Users/cygnus/src/matterhorn/dist-newstyle/build/matterhorn-0.1.0.0/build/autogen/Paths_matterhorn.hs, /Users/cygnus/src/matterhorn/dist-newstyle/build/matterhorn-0.1.0.0/build/matterhorn/matterhorn-tmp/Paths_matterhorn.o )
[ 3 of 34] Compiling Themes           ( src/Themes.hs, /Users/cygnus/src/matterhorn/dist-newstyle/build/matterhorn-0.1.0.0/build/matterhorn/matterhorn-tmp/Themes.o )
[ 4 of 34] Compiling Zipper           ( src/Zipper.hs, /Users/cygnus/src/matterhorn/dist-newstyle/build/matterhorn-0.1.0.0/build/matterhorn/matterhorn-tmp/Zipper.o )
[ 5 of 34] Compiling FilePaths        ( src/FilePaths.hs, /Users/cygnus/src/matterhorn/dist-newstyle/build/matterhorn-0.1.0.0/build/matterhorn/matterhorn-tmp/FilePaths.o )
[ 6 of 34] Compiling IOUtil           ( src/IOUtil.hs, /Users/cygnus/src/matterhorn/dist-newstyle/build/matterhorn-0.1.0.0/build/matterhorn/matterhorn-tmp/IOUtil.o )
[ 7 of 34] Compiling InputHistory     ( src/InputHistory.hs, /Users/cygnus/src/matterhorn/dist-newstyle/build/matterhorn-0.1.0.0/build/matterhorn/matterhorn-tmp/InputHistory.o )
<command line>: can't load .so/.DLL for: /Users/cygnus/src/matterhorn/dist-newstyle/build/mattermost-api-0.2.0.0/build/libHSmattermost-api-0.2.0.0-inplace-ghc7.10.3.dylib (dlopen(/Users/cygnus/src/matterhorn/dist-newstyle/build/mattermost-api-0.2.0.0/build/libHSmattermost-api-0.2.0.0-inplace-ghc7.10.3.dylib, 5): Symbol not found: _Cx2DfuJH59HGaWjhXQ60ZZZZ_Pathszumattermostzuapi_version1_closure
  Referenced from: /Users/cygnus/src/matterhorn/dist-newstyle/build/mattermost-api-0.2.0.0/build/libHSmattermost-api-0.2.0.0-inplace-ghc7.10.3.dylib
  Expected in: flat namespace
 in /Users/cygnus/src/matterhorn/dist-newstyle/build/mattermost-api-0.2.0.0/build/libHSmattermost-api-0.2.0.0-inplace-ghc7.10.3.dylib)
cygnus@plasma:~/src/matterhorn$ ghc --version
The Glorious Glasgow Haskell Compilation System, version 7.10.3
```

**Comment by jtdaugherty** [comment]
(This is with both `dist` and `dist-newstyle` being removed in advance of the test.)

**Comment by jtdaugherty** [comment]
Attn @aisamanra

**Comment by aisamanra** [comment]
This was because the `Paths_` module in `mattermost-api` didn't get included in `other-modules`. This is fixed by matterhorn-chat/mattermost-api@1a96041.
